modules/checkout/src/Plugin/Commerce/CheckoutFlow/CheckoutFlowBase.php

```diff
@@ -248,11 +248,10 @@ public function buildForm(array $form, FormStateInterface $form_state, $step_id
     $form['#attached']['library'][] = 'commerce_checkout/form';
     if ($this->hasSidebar($step_id)) {
       $form['sidebar']['order_summary'] = [
-        '#type' => 'view',
-        '#name' => 'commerce_checkout_order_summary',
-        '#display_id' => 'default',
-        '#arguments' => [$this->order->id()],
-        '#embed' => TRUE,
+        '#theme' => 'commerce_checkout_order_summary',
+        '#order' => $this->order,
```

Comment:
#order was renamed to #order_entity, but neither usage was updated, so the current code crashes.